extensions/quickgpt/src/contentFormat.ts

```diff
@@ -0,0 +1,59 @@
+type SpecificReplacements = {
+  query?: string;
+  clipboard?: string;
+  selection?: string;
+};
+
+const literalPlaceholder = {
+  query: "<输入文本>",
+  clipboard: "<剪贴板文本>",
+  selection: "<选中文本>",
+};
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English.